src/narada/window.py

```diff
@@ -88,6 +89,13 @@ async def dispatch_request(
         generate_gif: bool | None = None,
         output_schema: type[BaseModel] | None = None,
         timeout: int = 120,
+        previous_request_id: str | None = None,
```

Comment:
1. move the new fields before timeout
2. we need to add these fields to the two `@overload`s as well